dataloader.py

```diff
@@ -58,42 +86,51 @@ def PadSequence(batch):
     It will be passed to DataLoader as the keyword argument of collate_fn.
     '''
```

Comment:
Definitely add a comment here about what datatype of batch is. Would help clarify what happened in the rest of the function.